salt/_modules/metalk8s_kubernetes.py

```diff
@@ -893,26 +893,48 @@ def delete_service(name, namespace='default', **kwargs):
 
     CLI Examples::
 
-        salt '*' kubernetes.delete_service my-nginx default
-        salt '*' kubernetes.delete_service name=my-nginx namespace=default
+        salt '*' kubernetes.delete_service my-nginx-svc
+        salt '*' kubernetes.delete_service name=my-nginx-svc namespace=default
     '''
     cfg = _setup_conn(**kwargs)
+    body = kubernetes.client.V1DeleteOptions()
 
     try:
-        api_instance = kubernetes.client.CoreV1Api()
-        body = kubernetes.client.V1DeleteOptions()
+        api_instance = kubernetes.client.kubernetes.client.CoreV1Api()
         api_response = api_instance.delete_namespaced_service(
             name=name,
-            body=body,
-            namespace=namespace)
-
-        return api_response.to_dict()
+            namespace=namespace,
+            body=body)
+        mutable_api_response = api_response.to_dict()
+        if not salt.utils.platform.is_windows():
```

Comment:
Do we really want to support Windows…?

Reply:
idk, salt wants to keep a cross platform code for the modules, maybe we want to keep it Xplatform too

Reply:
We are already not cross platform for volumes (and probably at other places), so why bother here?

I'm not fan of adding complexity for a platform we will probably never supports.